### PR TITLE
New instrument cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Cal csvs for instruments returning from vendor refurb

### CODE REVIEW

File: calibration/OPTAAD/CGINS-OPTAAD-00185__20210622__CC_tcarray.ext [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+-0.001981,-0.000938,-0.000699,-0.000529,-0.000481,-0.000557,-0.000627,-0.000602,-0.000805,-0.000643,-0.000718,-0.000509,-0.000497,-0.000325,-0.000301,-0.000418,-5.8e-05,6.2e-05,0.000196,0.000164,0.000155,0.000209,0.000181,0.000176,7.1e-05,0,7e-05,-9.2e-05,-9.8e-05,-0.000342,-0.000477,-0.000499,-0.000852,-0.000945,-0.000907,-0.000929
+-0.001976,-0.00111,-0.000746,-0.000658,-0.00077,-0.000546,-0.000662,-0.000743,-0.000846,-0.000822,-0.000733,-0.000639,-0.000518,-0.000343,-0.00024,-0.000228,-8.1e-05,9.7e-05,0.000352,0.000302,0.000375,0.000435,0.000177,0.000419,0.000284,0,0.000327,0.000115,-8.7e-05,-4.6e-05,-0.000346,-0.000247,-0.000476,-0.000594,-0.00075,-0.000855
+-0.00242,-0.001647,-0.001336,-0.001021,-0.001129,-0.001189,-0.001323,-0.00117,-0.001174,-0.001401,-0.001296,-0.000988,-0.000987,-0.000792,-0.000751,-0.000668,-0.000568,-0.000227,-0.000132,1.1e-05,-7.7e-05,-2.7e-05,0.000316,0.000204,0.000232,0,0.000146,0.000116,2.9e-05,-9e-05,-0.000216,-0.000328,-0.000639,-0.000804,-0.00062,-0.000364
```
Comment (snwhite) [inline calibration/OPTAAD/CGINS-OPTAAD-00185__20210622__CC_tcarray.ext:84]: Looks good (spot checked).

File: calibration/OPTAAD/CGINS-OPTAAD-00185__20210622__CC_taarray.ext [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+0.001755,0.002718,0.003537,0.003604,0.003602,0.003555,0.003591,0.003465,0.003314,0.003263,0.003208,0.003089,0.002901,0.002803,0.002648,0.002541,0.00215,0.001942,0.001738,0.001376,0.001263,0.00098,0.000807,0.000583,0.000238,0,-0.000167,-0.000314,-0.00064,-0.000723,-0.000782,-0.00081,-0.001019,-0.000841,-0.000912,-0.000866
+0.001793,0.002722,0.003503,0.003602,0.003614,0.003511,0.003467,0.00347,0.003303,0.00331,0.003218,0.003132,0.003011,0.002829,0.002611,0.002574,0.002093,0.002032,0.001748,0.001405,0.001273,0.000939,0.000762,0.000579,0.00025,0,-0.000203,-0.000272,-0.000595,-0.00069,-0.000754,-0.000755,-0.000872,-0.000839,-0.000894,-0.001103
+0.001646,0.002585,0.003413,0.003525,0.003559,0.003507,0.003449,0.00344,0.003265,0.003274,0.003182,0.003106,0.002925,0.002885,0.002658,0.002515,0.002129,0.002033,0.001765,0.001374,0.001318,0.000976,0.000783,0.000564,0.000194,0,-0.000207,-0.000312,-0.000693,-0.000668,-0.000735,-0.000782,-0.000969,-0.000854,-0.000844,-0.000642
```
Comment (snwhite) [inline calibration/OPTAAD/CGINS-OPTAAD-00185__20210622__CC_taarray.ext:84]: Looks good (spot checked).

File: calibration/OPTAAD/CGINS-OPTAAD-00185__20210622.csv [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+ACS-185,CC_tbins,"[0.756189, 1.311881, 2.436875, 3.475217, 4.493750, 5.544091, 6.500417, 7.505200, 8.512500, 9.495000, 10.480800, 11.503600, 12.512308, 13.502083, 14.495833, 15.490417, 16.490385, 17.489615, 18.504783, 19.508800, 20.501250, 21.496000, 22.579500, 23.479600, 24.493077, 25.497692, 26.509231, 27.497241, 28.498333, 29.506800, 30.496071, 31.484444, 32.476296, 33.518519, 34.506429, 35.000000]",
+ACS-185,CC_taarray,SheetRef:CC_taarray,
+ACS-185,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (snwhite) [inline calibration/OPTAAD/CGINS-OPTAAD-00185__20210622.csv:9]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00242__20210512.csv [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+ACS-242,CC_tbins,"[0.782453, 1.264380, 2.461250, 3.467917, 4.531333, 5.481667, 6.500417, 7.501200, 8.512400, 9.502500, 10.481667, 11.474400, 12.476800, 13.472500, 14.495833, 15.507200, 16.512083, 17.491739, 18.482308, 19.499231, 20.506154, 21.493200, 22.493600, 23.502800, 24.491538, 25.514000, 26.487083, 27.525172, 28.479583, 29.478400, 30.503462, 31.496087, 32.472308, 33.507143, 34.457619]",
+ACS-242,CC_taarray,SheetRef:CC_taarray,
+ACS-242,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (snwhite) [inline calibration/OPTAAD/CGINS-OPTAAD-00242__20210512.csv:9]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00242__20210512__CC_tcarray.ext [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+-0.010539,-0.009054,-0.00824,-0.007571,-0.006427,-0.007174,-0.007198,-0.006731,-0.00627,-0.005954,-0.005565,-0.005232,-0.004683,-0.004284,-0.003876,-0.003504,-0.003034,-0.002681,-0.002258,-0.001881,-0.001416,-0.001045,-0.000761,-0.00045,-0.00024,0,0.000114,0.000251,0.000476,0.000544,0.000693,0.000675,0.000871,0.000784,0.000641
+-0.01116,-0.009489,-0.008171,-0.007496,-0.006289,-0.006982,-0.006963,-0.00653,-0.006187,-0.005704,-0.005402,-0.004948,-0.004505,-0.004264,-0.003807,-0.003404,-0.003157,-0.002702,-0.002329,-0.001867,-0.001528,-0.001203,-0.000751,-0.00044,-0.000268,0,0.00027,0.000405,0.000584,0.000702,0.000849,0.000859,0.000874,0.000877,0.000607
+-0.010938,-0.008409,-0.007736,-0.007359,-0.006328,-0.007087,-0.007171,-0.006703,-0.00626,-0.005805,-0.005275,-0.004884,-0.004469,-0.004007,-0.003673,-0.003111,-0.00265,-0.002369,-0.001907,-0.001856,-0.001333,-0.001142,-0.000561,-0.000407,-0.000139,0,0.000381,0.000475,0.00063,0.000736,0.000922,0.000976,0.000978,0.000948,0.000546
```
Comment (snwhite) [inline calibration/OPTAAD/CGINS-OPTAAD-00242__20210512__CC_tcarray.ext:77]: Looks good (spot checked).

File: calibration/OPTAAD/CGINS-OPTAAD-00242__20210512__CC_taarray.ext [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+-0.009888,-0.00779,-0.006649,-0.006284,-0.005944,-0.005788,-0.005525,-0.005157,-0.004865,-0.004518,-0.004284,-0.003925,-0.003604,-0.003328,-0.003034,-0.002745,-0.00244,-0.002187,-0.002007,-0.001566,-0.001322,-0.001101,-0.000753,-0.00052,-0.000236,0,0.000319,0.000406,0.000608,0.000814,0.000795,0.000872,0.001065,0.0011,0.000978
+-0.010123,-0.007991,-0.006854,-0.00645,-0.006076,-0.006004,-0.005699,-0.005259,-0.005042,-0.004695,-0.004399,-0.004109,-0.003706,-0.003464,-0.003098,-0.002915,-0.002523,-0.002295,-0.002096,-0.001657,-0.001376,-0.001123,-0.000768,-0.000578,-0.000276,0,0.000305,0.000432,0.00062,0.000879,0.000875,0.001056,0.00109,0.001126,0.001023
+-0.010354,-0.008215,-0.007092,-0.006666,-0.006349,-0.006135,-0.005854,-0.005458,-0.00521,-0.004788,-0.004538,-0.004178,-0.003839,-0.003592,-0.003243,-0.002987,-0.002584,-0.002391,-0.002171,-0.001743,-0.00147,-0.00114,-0.000828,-0.000581,-0.000213,0,0.000347,0.000483,0.000695,0.000947,0.001006,0.001144,0.00123,0.001276,0.00121
```
Comment (snwhite) [inline calibration/OPTAAD/CGINS-OPTAAD-00242__20210512__CC_taarray.ext:77]: Looks good (spot checked).

File: calibration/OPTAAD/CGINS-OPTAAD-00255__20210527.csv [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+ACS-255,CC_tbins,"[0.645931, 1.400313, 2.477667, 3.487619, 4.561176, 5.489565, 6.492800, 7.504000, 8.495600, 9.488400, 10.487500, 11.484231, 12.511200, 13.501667, 14.502000, 15.504800, 16.486667, 17.489200, 18.481667, 19.490385, 20.492500, 21.504000, 22.494000, 23.488000, 24.487692, 25.490000, 26.490000, 27.503333, 28.512963, 29.503462, 30.511200, 31.487600, 32.500385, 33.533448, 34.486800, 35.000000]",
+ACS-255,CC_taarray,SheetRef:CC_taarray,
+ACS-255,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (snwhite) [inline calibration/OPTAAD/CGINS-OPTAAD-00255__20210527.csv:9]: . (spot checked).

File: calibration/OPTAAD/CGINS-OPTAAD-00255__20210527__CC_taarray.ext [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+-0.006464,-0.004373,-0.003559,-0.00326,-0.003146,-0.002957,-0.002734,-0.002291,-0.002436,-0.002036,-0.001839,-0.001751,-0.001629,-0.001476,-0.001244,-0.001289,-0.00094,-0.00079,-0.000566,-0.000522,-0.000291,-0.000159,-0.000115,-8e-05,-7.3e-05,0,3.3e-05,-2.1e-05,4.3e-05,-0.000166,-0.000416,-0.000432,-0.000527,-0.000711,-0.000712,-0.000423
+-0.006865,-0.004817,-0.003992,-0.003589,-0.003579,-0.003434,-0.003067,-0.002806,-0.00273,-0.002393,-0.002143,-0.001975,-0.001837,-0.001676,-0.001412,-0.001459,-0.001026,-0.000797,-0.00061,-0.000699,-0.000385,-0.000294,-9.2e-05,-0.000122,-7.2e-05,0,3.6e-05,0.000103,0.000159,-4.8e-05,-0.000352,-0.00031,-0.000453,-0.000486,-0.000447,-0.000478
+-0.007174,-0.005136,-0.004223,-0.003859,-0.00371,-0.003562,-0.003293,-0.002877,-0.003046,-0.002601,-0.002471,-0.002258,-0.002096,-0.001868,-0.001559,-0.001656,-0.001131,-0.001035,-0.000871,-0.000743,-0.000316,-0.000317,-8.3e-05,-7.5e-05,5.1e-05,0,0.000142,0.000122,0.000185,0.000108,-0.00017,-0.000154,-0.000322,-0.000302,-0.000424,-0.000886
```
Comment (snwhite) [inline calibration/OPTAAD/CGINS-OPTAAD-00255__20210527__CC_taarray.ext:85]: Looks good (spot checked).

File: calibration/OPTAAD/CGINS-OPTAAD-00255__20210527__CC_tcarray.ext [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+0.005104,0.003822,0.003598,0.00336,0.003053,0.002682,0.002392,0.002233,0.002043,0.001428,0.0012,0.000644,0.000357,0.000163,-0.000151,-0.000451,-0.000247,7.9e-05,-0.000444,-0.00043,-0.000528,-8.2e-05,-0.000213,-7.3e-05,-0.0001,0,-0.000241,-0.000392,-0.000125,-0.000662,-0.000685,-0.000615,-0.000642,-0.001298,-0.001511,-0.001067
+0.004635,0.00324,0.002953,0.002424,0.002532,0.001873,0.001689,0.001474,0.001493,0.001283,0.001214,0.00117,0.000831,0.000596,2.2e-05,-0.00025,-7.7e-05,0.000169,-0.000206,-0.000378,-0.000518,-0.000208,-0.000165,-2e-05,-4e-06,0,-0.000424,-0.000381,1.7e-05,-0.000693,-0.000662,-0.000328,-0.000479,-0.000851,-0.001006,-0.001608
+0.004167,0.003276,0.003159,0.002788,0.002634,0.001943,0.001631,0.001523,0.001207,0.000705,0.00063,0.000583,0.000438,0.000187,-0.000124,-0.000334,-1.5e-05,0.000247,-0.000525,-0.000365,-0.000312,8.6e-05,-0.000207,-0.000163,-0.000176,0,-0.000271,-0.000633,-0.000269,-0.00047,-0.000669,-0.000797,-0.000579,-0.000999,-0.001405,-0.002133
```
Comment (snwhite) [inline calibration/OPTAAD/CGINS-OPTAAD-00255__20210527__CC_tcarray.ext:85]: Looks good (spot checked).

**Review by snwhite — APPROVED** [review]

**Comment by snwhite** [comment]
[PYII Instrument Cal CSVs [DT-69]](https://trello.com/c/CbhVKyjb/69-pyii-instrument-cal-csvs-dt-69)

File: calibration/FLORTD/CGINS-FLORTD-01214__20210706.csv [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+1214,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1214_Calibration_Files_2021-07-06.zip > BBFL2W-1214 CharSheets.pdf
+1214,CC_dark_counts_cdom,42,
+1214,CC_dark_counts_chlorophyll_a,51,
+1214,CC_dark_counts_volume_scatter,50,
+1214,CC_depolarization_ratio,0.039,Constant
+1214,CC_measurement_wavelength,700,Constant
+1214,CC_scale_factor_cdom,0.0621,
+1214,CC_scale_factor_chlorophyll_a,0.0121,
+1214,CC_scale_factor_volume_scatter,2.350e-06,
+1214,CC_scattering_angle,124,Constant
```
Comment (cdobs) [inline calibration/FLORTD/CGINS-FLORTD-01214__20210706.csv:11]: Looks good

File: calibration/FLORTD/CGINS-FLORTD-01215__20210706.csv [hunk 11]
```diff
@@ -0,0 +1,11 @@
...
+1215,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1215_Calibration_Files_2021-07-06.zip > BBFL2W-1215 Charsheets.pdf and BBFL2W-1215 Chlorophyll Charsheet.pdf
+1215,CC_dark_counts_cdom,45,
+1215,CC_dark_counts_chlorophyll_a,52,
+1215,CC_dark_counts_volume_scatter,50,
+1215,CC_depolarization_ratio,0.039,Constant
+1215,CC_measurement_wavelength,700,Constant
+1215,CC_scale_factor_cdom,0.0552,
+1215,CC_scale_factor_chlorophyll_a,0.0104,
+1215,CC_scale_factor_volume_scatter,2.217e-06,
+1215,CC_scattering_angle,124,Constant
```
Comment (cdobs) [inline calibration/FLORTD/CGINS-FLORTD-01215__20210706.csv:11]: Looks good

File: calibration/FLORTD/CGINS-FLORTD-01216__20210708.csv [hunk 12]
```diff
@@ -0,0 +1,11 @@
...
+1216,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1216_Calibration_Files_2021-07-08.zip > BBFL2W-1216_CharSheets.pdf
+1216,CC_dark_counts_cdom,46,
+1216,CC_dark_counts_chlorophyll_a,45,
+1216,CC_dark_counts_volume_scatter,46,
+1216,CC_depolarization_ratio,0.039,Constant
+1216,CC_measurement_wavelength,700,Constant
+1216,CC_scale_factor_cdom,0.0894,
+1216,CC_scale_factor_chlorophyll_a,0.0123,
+1216,CC_scale_factor_volume_scatter,2.211e-06,
+1216,CC_scattering_angle,124,Constant
```
Comment (cdobs) [inline calibration/FLORTD/CGINS-FLORTD-01216__20210708.csv:11]: Looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01103__20210707.csv [hunk 13]
```diff
@@ -0,0 +1,8 @@
...
...272.64, 273.44, 274.25, 275.05, 275.85, 276.66, 277.46, 278.26, 279.07, 279.87, 280.67, 281.48, 282.28, 283.08, 283.89, 284.69, 285.49, 286.3, 287.1, 287.91, 288.71, 289.51, 290.32, 291.12, 291.92, 292.73, 293.53, 294.33, 295.14, 295.94, 296.74, 297.55, 298.35, 299.15, 299.96, 300.76, 301.56, 302.37, 303.17, 303.97, 304.78, 305.58, 306.38, 307.19, 307.99, 308.79, 309.59, 310.4, 311.2, 312, 312.81, 313.61, 314.41, 315.21, 316.02, 316.82, 317.62, 318.42, 319.22, 320.03, 320.83, 321.63, 322.43, 323.23, 324.04, 324.84, 325.64, 326.44, 327.24, 328.04, 328.84, 329.64, 330.45, 331.25, 332.05, 332.85, 333.65, 334.45, 335.25, 336.05, 336.85, 337.65, 338.45, 339.25, 340.05, 340.85, 341.65, 342.45, 343.25, 344.04, 344.84, 345.64, 346.44, 347.24, 348.04, 348.84, 349.63, 350.43, 351.23, 352.03, 352.82, 353.62, 354.42, 355.22, 356.01, 356.81, 357.61, 358.4, 359.2, 360, 360.79, 361.59, 362.38, 363.18, 363.97, 364.77, 365.56, 366.36, 367.15, 367.95, 368.74, 369.53, 370.33, 371.12, 371.92, 372.71, 373.5, 374.29, 375.09, 375.88, 376.67, 377.46, 378.26, 379.05, 379.84, 380.63, 381.42, 382.21, 383, 383.79, 384.58, 385.37, 386.16, 386.95, 387.74, 388.53, 389.32, 390.11, 390.9, 391.68, 392.47, 393.26]",
```
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01103__20210707.csv:8]: Looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01107__20210611.csv [hunk 14]
```diff
@@ -0,0 +1,8 @@
...
...61, 272.42, 273.23, 274.05, 274.86, 275.67, 276.49, 277.3, 278.11, 278.93, 279.74, 280.55, 281.37, 282.18, 282.99, 283.81, 284.62, 285.43, 286.24, 287.06, 287.87, 288.68, 289.5, 290.31, 291.12, 291.93, 292.75, 293.56, 294.37, 295.18, 295.99, 296.81, 297.62, 298.43, 299.24, 300.05, 300.87, 301.68, 302.49, 303.3, 304.11, 304.92, 305.74, 306.55, 307.36, 308.17, 308.98, 309.79, 310.6, 311.41, 312.22, 313.03, 313.84, 314.65, 315.46, 316.27, 317.08, 317.89, 318.7, 319.51, 320.32, 321.13, 321.94, 322.75, 323.56, 324.37, 325.18, 325.98, 326.79, 327.6, 328.41, 329.22, 330.02, 330.83, 331.64, 332.45, 333.25, 334.06, 334.87, 335.68, 336.48, 337.29, 338.09, 338.9, 339.71, 340.51, 341.32, 342.12, 342.93, 343.73, 344.54, 345.34, 346.15, 346.95, 347.76, 348.56, 349.37, 350.17, 350.97, 351.78, 352.58, 353.38, 354.18, 354.99, 355.79, 356.59, 357.39, 358.2, 359, 359.8, 360.6, 361.4, 362.2, 363, 363.8, 364.6, 365.4, 366.2, 367, 367.8, 368.6, 369.4, 370.19, 370.99, 371.79, 372.59, 373.39, 374.18, 374.98, 375.78, 376.57, 377.37, 378.16, 378.96, 379.76, 380.55, 381.35, 382.14, 382.93, 383.73, 384.52, 385.32, 386.11, 386.9, 387.69, 388.49, 389.28, 390.07, 390.86, 391.65, 392.45, 393.24, 394.03, 394.82]",
```
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01107__20210611.csv:8]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00153__20210609.csv [hunk 15]
```diff
@@ -0,0 +1,9 @@
...
+ACS-153,CC_tbins,"[1.510763, 2.409400, 3.484615, 4.481667, 5.560000, 6.477200, 7.484800, 8.496154, 9.520385, 10.502800, 11.494615, 12.488077, 13.501600, 14.509583, 15.506667, 16.497600, 17.506000, 18.494167, 19.485600, 20.502308, 21.504583, 22.495200, 23.492800, 24.497200, 25.476923, 26.500769, 27.493462, 28.496800, 29.523462, 30.500000, 31.509200, 32.498750, 33.510345, 34.478966]",
+ACS-153,CC_taarray,SheetRef:CC_taarray,
+ACS-153,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00153__20210609.csv:9]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00153__20210609__CC_taarray.ext [hunk 16]
```diff
@@ -0,0 +1,85 @@
...
+-0.008352,-0.006171,-0.005347,-0.005191,-0.004875,-0.004563,-0.004184,-0.003737,-0.003456,-0.00313,-0.002823,-0.002638,-0.002379,-0.002038,-0.001785,-0.001685,-0.001257,-0.001223,-0.000894,-0.00079,-0.00061,-0.000494,-0.00034,-9.9e-05,0,8.2e-05,8.6e-05,6.4e-05,0.000249,0.000387,0.000398,0.000497,0.000471,0.000311
+-0.008903,-0.006702,-0.005935,-0.005564,-0.005336,-0.004887,-0.004505,-0.004016,-0.003812,-0.003417,-0.003077,-0.002912,-0.002576,-0.00219,-0.001999,-0.001865,-0.001448,-0.001354,-0.001081,-0.000932,-0.000636,-0.000569,-0.000398,-0.000136,0,8.1e-05,0.00015,0.000205,0.00034,0.000498,0.000539,0.000499,0.000624,0.000408
+-0.009076,-0.006871,-0.00607,-0.005768,-0.005474,-0.005055,-0.004634,-0.004217,-0.003908,-0.003484,-0.003174,-0.003041,-0.002753,-0.002386,-0.002124,-0.001988,-0.001494,-0.001439,-0.001069,-0.000958,-0.000707,-0.000515,-0.000417,-0.000141,0,0.000145,0.000107,0.000261,0.000365,0.000555,0.000603,0.000679,0.000666,0.000527
```
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00153__20210609__CC_taarray.ext:85]: Spot check - looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00153__20210609__CC_tcarray.ext [hunk 17]
```diff
@@ -0,0 +1,85 @@
...
+0.000973,0.001637,0.001585,0.001421,0.001313,0.00118,0.001183,0.001041,0.001046,0.001038,0.000924,0.000902,0.000894,0.000709,0.000679,0.000496,0.000449,0.000467,0.000414,0.000289,0.000287,0.000138,6.6e-05,0.000143,0,-0.000153,-0.000172,-0.000368,-0.0003,-0.000688,-0.000651,-0.000726,-0.000809,-0.001003
+0.000367,0.001019,0.001001,0.000928,0.000787,0.00072,0.000785,0.000644,0.000557,0.000606,0.000694,0.000565,0.000476,0.000554,0.000467,0.000314,0.000396,0.000349,0.000224,0.000267,0.000257,8.8e-05,-7e-06,6e-05,0,-8e-06,1e-05,-0.000115,-0.000245,-0.00028,-0.00031,-0.000514,-0.000654,-0.000728
+5.4e-05,0.000607,0.000763,0.000707,0.000664,0.00042,0.000435,0.000479,0.000475,0.000491,0.000255,0.00038,0.000519,0.000396,0.000346,0.000274,0.000202,5.6e-05,0.000242,0.000123,0.000196,0.0001,9.5e-05,0.000148,0,-4.5e-05,-0.000202,-0.000142,-0.000192,-0.000239,-0.000322,-0.000327,-0.000492,-0.000574
+-0.000413,0.000356,0.000282,0.000242,0.000153,0.000231,0.000195,4.9e-05,5.3e-05,0.000136,0.000163,0.000149,0.000126,0.000128,5.4e-05,-8.7e-05,3e-06,8e-05,1.1e-05,3.8e-05,0.000149,2.3e-05,-3e-06,4.5e-05,0,-2.4e-05,4e-05,-0.000267,-1e-05,-0.000411,-0.000263,-0.000458,-0.000378,-0.00052
```
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00153__20210609__CC_tcarray.ext:85]: Spot check - looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00185__20210622.csv (hunk 3 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00185__20210622.csv:9]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00185__20210622__CC_taarray.ext (hunk 2 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00185__20210622__CC_taarray.ext:84]: Spot check - looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00185__20210622__CC_tcarray.ext (hunk 1 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00185__20210622__CC_tcarray.ext:84]: Spot check - looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00242__20210512.csv (hunk 4 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00242__20210512.csv:9]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00242__20210512__CC_taarray.ext (hunk 6 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00242__20210512__CC_taarray.ext:77]: Spot check - looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00242__20210512__CC_tcarray.ext (hunk 5 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00242__20210512__CC_tcarray.ext:77]: Spot check - looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00255__20210527.csv (hunk 7 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00255__20210527.csv:9]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00255__20210527__CC_taarray.ext (hunk 8 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00255__20210527__CC_taarray.ext:85]: Spot check - looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00255__20210527__CC_tcarray.ext (hunk 9 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00255__20210527__CC_tcarray.ext:85]: Spot check - looks good

File: calibration/PCO2WB/CGINS-PCO2WB-C0098__20210706.csv [hunk 18]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0098,CC_cala,0.0095,source_file PCO2W-B_SAMI-CO2_SN_C0098_Calibration_Files_2021-07-06.zip > CalCert_C0098_July2021.pdf
+C0098,CC_calb,0.8195,
+C0098,CC_calc,-1.7807,
+C0098,CC_calt,14.5145,
+C0098,CC_ea434,19706,constant
+C0098,CC_ea620,34,constant
+C0098,CC_eb434,3073,constant
+C0098,CC_eb620,44327,constant
```
Comment (cdobs) [inline calibration/PCO2WB/CGINS-PCO2WB-C0098__20210706.csv:9]: Looks good

File: calibration/PCO2WB/CGINS-PCO2WB-C0100__20210802.csv [hunk 19]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0100,CC_cala,0.0803,source_file PCO2W-B_SAMI-CO2_SN_C0100_Calibration_Files_2021-08-02.zip > SAMI_010821_CalCert.pdf
+C0100,CC_calb,0.4268,
+C0100,CC_calc,-1.2284,
+C0100,CC_calt,6.7484,
+C0100,CC_ea434,19706,constant
+C0100,CC_ea620,34,constant
+C0100,CC_eb434,3073,constant
+C0100,CC_eb620,44327,constant
```
Comment (cdobs) [inline calibration/PCO2WB/CGINS-PCO2WB-C0100__20210802.csv:9]: Looks good

File: calibration/PCO2WB/CGINS-PCO2WB-C0134__20210706.csv [hunk 20]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0134,CC_cala,0.0424,source_file PCO2W-B_SAMI-CO2_SN_C0134_Calibration_Files_2021-07-06.zip > CalCert_C0134_July2021.pdf
+C0134,CC_calb,0.6712,
+C0134,CC_calc,-1.5912,
+C0134,CC_calt,14.5352,
+C0134,CC_ea434,19706,constant
+C0134,CC_ea620,34,constant
+C0134,CC_eb434,3073,constant
+C0134,CC_eb620,44327,constant
```
Comment (cdobs) [inline calibration/PCO2WB/CGINS-PCO2WB-C0134__20210706.csv:9]: Looks good

File: calibration/PHSEND/CGINS-PHSEND-P0095__20210716.csv [hunk 21]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0095,CC_ea434,17372.0, source_file PHSEN-D_SAMI-pH_SN_P0095_Validation_Files_2021-07-16.zip > Val_cert_P0095.pdf
+P0095,CC_ea578,94.1,
+P0095,CC_eb434,2284.0,
+P0095,CC_eb578,38676.0,
+P0095,CC_ind_off,0,
+P0095,CC_ind_slp,1,
+P0095,CC_psal,35,Constant
```
Comment (cdobs) [inline calibration/PHSEND/CGINS-PHSEND-P0095__20210716.csv:8]: Looks good

File: calibration/SPKIRB/CGINS-SPKIRB-00274__20210420.csv [hunk 22]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+274,CC_immersion_factor,"[1.368, 1.410, 1.365, 1.354, 1.372, 1.322, 1.347] ", source_file SPKIR-B_OCR-507_SN_274_Calibration_Files_2021-04-20.zip > DI7274E.CAL
+274,CC_offset,"[2147567510.0, 2148022491.0, 2147639995.7, 2147506159.6, 2147840470.8, 2147921318.0, 2147019447.1] ",
+274,CC_scale,"[2.02525307961e-007, 1.99003581252e-007, 2.05045340332e-007, 2.0711991949e-007, 2.11912677683e-007, 2.04708063507e-007, 2.07545475112e-007] ",
```
Comment (cdobs) [inline calibration/SPKIRB/CGINS-SPKIRB-00274__20210420.csv:4]: Looks good

**Review by cdobs — APPROVED** [review]
Everything looks good

**Pull request merged**